src/pages/scientificServices/pipelines/components/inputs/PipelineFloatInput.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { ValidatedInput } from 'src/components/input';
+import { PipelineInput } from 'src/libs/ajax/teaspoons/teaspoons-models';
+import { INPUT_DESCRIPTIONS } from 'src/pages/scientificServices/pipelines/utils/pipeline-input-utils';
+
+interface PipelineFloatInputProps {
+  input: PipelineInput;
+  value: string;
+  onChange: (value: string) => void;
+  validationError?: string;
+  onValidation(error?: string): void;
+}
+
+export const PipelineFloatInput: React.FC<PipelineFloatInputProps> = ({
+  input,
+  value,
+  onChange,
+  validationError,
+  onValidation,
+}) => {
+  const { label, placeholder, helpText, validationRegex } = INPUT_DESCRIPTIONS[input.name] || {};
+
+  return (
+    <>
+      <h3 style={{ marginBottom: '0.5rem' }}>
+        {label || input.name} {input.isRequired ? <span style={{ color: '#DB3214' }}> *</span> : null}
+      </h3>
+      <ValidatedInput
+        width={400}
+        error={validationError}
+        inputProps={{
+          'aria-label': `${input.name} float input`,
+          type: 'text',
+          value: value || '',
+          placeholder: placeholder || '',
+          onChange: (e) => {
+            onChange(e);
+            if (validationRegex) {
+              const regex = new RegExp(validationRegex);
+              if (!regex.test(e.trim()) && e.length > 0) {
+                onValidation('Invalid float value');
+              } else {
+                onValidation(undefined);
+              }
+            }
+          },
+        }}
+      />
+      {helpText && <div style={{ marginTop: '0.5rem', marginBottom: '2rem', fontStyle: 'italic' }}>{helpText}</div>}
```

Comment:
i think we should put a maxwidth on this div, so the text wraps after a certain point:


```suggestion
      {helpText && <div style={{ marginTop: '0.5rem', marginBottom: '2rem', fontStyle: 'italic', maxWidth: 500 }}>{helpText}</div>}
```